criu/bpfmap.c

```diff
@@ -270,16 +280,21 @@ const struct fdtype_ops bpfmap_dump_ops = {
 
 static int bpfmap_open(struct file_desc *d, int *new_fd)
 {
-	struct bpfmap_file_info *info;
-	BpfmapFileEntry *bpfe;
+	struct bpfmap_file_info *info = container_of(d, struct bpfmap_file_info, d);
+	BpfmapFileEntry *bpfe = info->bpfe;
+	struct bpf_create_map_attr xattr = {
+		.name = strdup(bpfe->map_name),
```

Comment:
`xstrdup`

Reply:
Also, I think we should move the initialization block after all local variable declarations